blocks/library/video/style.scss

```diff
@@ -1,6 +1,14 @@
-.wp-block-video figcaption {
-	margin-top: 0.5em;
-	color: $dark-gray-300;
-	text-align: center;
-	font-size: $default-font-size;
+.wp-block-video {
+	margin: 0;
+
+	figcaption {
```

Comment:
It'd be nice to not repeat these styles over and over. Maybe we should have a `Caption` component that people can use? cc @aduth @youknowriad

Reply:
Thi sis not an editor style, if we create a caption component with its own styles, these will be included in the editor only and not in the frontend. 

We could probably update our webpack config or move this component to a location where its styles get loaded in the frontend as well, but we'd have to update the `save` functions of all these blocks to use this new component as well which will modify (and invalid) these blocks because of the component's className (unless we want to target `figcaption` globally but it seems dangerous)

Reply:
Can I merge this as is? The discussion here can continue, but wondering if I should ship this as is in the meantime.